[tox]
envlist =
    py27,
    py34,
    py35,
    py36,
# pypy fails because of some weird test error that might be an issue with
# sqlalchemy, pypy, and issues with object identity.

[testenv]
deps =
    .[test]
    zope.testrunner
commands =
    zope-testrunner --test-path=src {posargs:-pvc}
